src/shared/components/case-editor/case-edit-form/case-edit-form.html

```diff
@@ -1,6 +1,6 @@
-<ng-container *ngFor="let field of fields">
-  <div ccdConditionalShow ccdLabelSubstitutor
-       [caseField]="field" [formGroup]="formGroup" [contextFields]="caseFields" [greyBarEnabled]="true">
+<ng-container ccdConditionalShowForm [formGroup]="formGroup" [caseFields]="fields" [contextFields]="caseFields" *ngFor="let field of fields">
```

Comment:
the greyBarEnabled is removed, is that set to true by default it implements in the new ccdConditionalShowForm?

Reply:
Gray bar not currently working, but is not enabled on a per field basis